ts/store/reducers/entities/services/servicesByReadStatus.ts

```diff
@@ -0,0 +1,43 @@
+import * as pot from "italia-ts-commons/lib/pot";
+import { getType } from "typesafe-actions";
+import { ServicePublic } from "../../../../../definitions/backend/ServicePublic";
+import { clearCache } from "../../../actions/profile";
+import { showServiceDetails } from "../../../actions/services";
+import { Action } from "../../../actions/types";
+import { GlobalState } from "../../types";
+
+export type ServicesByReadState = Readonly<{
```

Comment:
ServicesByOrganizationId mean an object with organizationId as key and services as value.
For the same logic ServicesByReadState mean an object with readState as key and services as value, and it is not your case. Find an appropriate name.
Also is there a reason to store the whole Pot object as value? Isn't "true" enough?

Reply:
Fix in 164293d